tests/SprykerSdkTest/Zed/AppSdk/Communication/Console/CreateManifestConsoleTest.php

```diff
@@ -8,6 +8,7 @@
 namespace SprykerSdkTest\Zed\AppSdk\Communication\Console;
 
 use Codeception\Test\Unit;
+use org\bovigo\vfs\vfsStream;
```

Comment:
This line can be removed. These things belong to the `// Arrange` section and will only be used inside of a Helper.

In arrange you will call something like `$this->tester->have...()`. With that, we keep our tests clean and make code re-usable in other tests as well.